en/appendices/3-2-migration-guide.rst

```diff
@@ -60,7 +76,14 @@ a new ``CorsBuilder`` has been added. This class lets you define CORS related
 headers with a fluent interface. See :ref:`cors-headers` for more information.
 
 ORM
----
+===
 
 * Containing the same association multiple times now works as expected, and the
   query builder functions are now stacked.
+
+
+Shell
+=====
+
+* ``Shell::info()``, ``Shell::warn()`` and ``Shell::success()`` were added.
+  These helper methods make using commonly used styling simpler.
```

Comment:
Is it worth noting that warn() goes to stderr instead - compared to the other two.